### PR TITLE
Add misspell: countermeasure, kernel, pthread, thermistor

### PR DESCRIPTION
**Pull request by vikivivi**
* New entry words: counterfeits, countermeasure, countermeasures, counterparts, coverages, thermistor, thermistors
* Typical code search:
  * https://github.com/search?q=countermeausures&type=code
  * https://github.com/search?q=coutermeasueres&type=code
  * https://github.com/search?q=coutermeasures&type=code
  * https://github.com/search?q=kernerl&type=code
  * https://github.com/search?q=phtread&type=code
  * https://github.com/search?q=thermisor&type=code

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -7502,18 +7507,26 @@ couses->courses, causes,
...
 couuldn't->couldn't
 covarage->coverage
+covarages->coverages
 covarege->coverage
 covention->convention
+coventions->conventions
 covere->cover
 coveres->covers
 coverge->coverage
+coverges->coverages
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:7529]: Suggest converges too

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -7502,18 +7507,26 @@ couses->courses, causes,
...
 couuld->could
 couuldn't->couldn't
 covarage->coverage
+covarages->coverages
 covarege->coverage
 covention->convention
+coventions->conventions
 covere->cover
 coveres->covers
 coverge->coverage
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:7528]: Suggest converge too

**Review by peternewman — CHANGES_REQUESTED** [review]
A few improvements please

**Comment by vikivivi** [comment]
@peternewman As per your change request, both `converge` and `converges` are added to suggestion.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -7502,18 +7507,26 @@ couses->courses, causes,
 cousing->cousin
 couted->counted
 couter->counter
+coutermeasuere->countermeasure
+coutermeasueres->countermeasures
+coutermeasure->countermeasure
+coutermeasures->countermeasures
 couterpart->counterpart
+couters->counters
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:7515]: Suggest routers and scouters, the latter is possibly a bit niche but hopefully okay
```suggestion
couters->counters, routers, scouters,
```

**Review by peternewman — CHANGES_REQUESTED** [review]
Sorry @vikivivi , one more comment

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Thanks

**Review by peternewman — APPROVED** [review]
Thanks @vikivivi

**Pull request merged**